.github/workflows/publish.yaml

```
@@ -63,3 +64,16 @@ jobs:
           docker build --cache-from $API_IMAGE --target prod --tag $API_IMAGE ./api
           docker tag $API_IMAGE $API_IMAGE:${{ inputs.image-tag }}
           docker push $API_IMAGE:${{ inputs.image-tag }}
+
+  build-and-publish-nginx:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: "Build and Publish nginx"
+        run: |
+          echo "Tagging with ${{ inputs.image-tag }}"
+          echo ${{ secrets.GITHUB_TOKEN }} | docker login $IMAGE_REGISTRY -u $GITHUB_ACTOR --password-stdin
+          # docker pull $NGINX_IMAGE
+          docker build --cache-from $NGINX_IMAGE --tag $NGINX_IMAGE ./nginx
```

```suggestion
          docker pull $NGINX_IMAGE
          docker build --cache-from $NGINX_IMAGE --tag $NGINX_IMAGE ./nginx
```
Has this been tested?
I believe it might fail, at least on the initial run, due to attempting to cache from the `$NGINX_IMAGE` which at (initial[?]) runtime is not present (due to the `pull` command being commented out)

---

It failed on docker pull, since the nginx image did not exist. Now it's exist, so can un-comment it again. The docker build does not fail, even if docker pull is not called.

---

Yeah, maybe I remembered it the other way around! All good then.